### PR TITLE
[Readme] fixing code formatting typo in main Ubuntu readme

### LINKED ISSUE #5424: README not rendering properly on master branch
There is an issue with the Markdown formatting (scroll to the bottom).

### PR DESCRIPTION
**Pull request by christinerogers**
Fixing a minor formatting issue in the Ubuntu readme
This resolves #5424 reported by @johnsaigle

This is issued to master not 22-release since 22-release's readme isn't quite up to date

Testing instructions 

- [ ]  View the Readme on Github. check if it renders properly.

### CODE REVIEW

**Comment by ridz1208** [comment]
@christinerogers I dont believe there will be a master release antimesoon
is the problem there on the 21.0-release branch ? you could issue the PR there (that branch is up to date with 21.0.3 release and will become the 21.0.4 release)

**Comment by johnsaigle** [comment]
@ridz1208 The problem is on `master`, `21.0-release` and `22.0-release` branches. I created the original issue before `22.0-release` was created.

This can be rebased and put into the release.

**Comment by ridz1208** [comment]
rebase to 22 please

**Comment by johnsaigle** [comment]
Bumping for rebase

File: README.md [hunk 1]
```diff
@@ -75,11 +75,13 @@ Consult the [LORIS Wiki](https://github.com/aces/Loris/wiki/Setup) page on this
 5. Apache configuration
 
 If your apache configuration was not completed by the Install script, run the following enable rewriting of LORIS, enable your `$projectname` site, and restart apache:  (run by user who has root privileges)
-```bash
+    ```bash
```
Comment (johnsaigle) [inline README.md]:
```suggestion
    
```bash
```

**Comment by johnsaigle** [comment]
@ridz1208 @driusan This isn't "Critical" exactly but the README on 22.0-release looks really weird without it.

**Comment by christinerogers** [comment]
Commit accepted, thanks @johnsaigle 

Ready to merge upon re-review @ridz1208  @driusan

File: README.md [hunk 2]
```diff
@@ -75,11 +75,14 @@ Consult the [LORIS Wiki](https://github.com/aces/Loris/wiki/Setup) page on this
 5. Apache configuration
 
 If your apache configuration was not completed by the Install script, run the following enable rewriting of LORIS, enable your `$projectname` site, and restart apache:  (run by user who has root privileges)
+    
 ```bash
     sudo a2enmod rewrite
     sudo a2dissite default
     sudo a2ensite $projectname
     sudo service apache2 reload
+    ```
```
Comment (johnsaigle) [inline README.md]: I think this has to be at the same indentation level as the "bash" line

Comment (christinerogers) [reply README.md]: ok.  just committed again.  re-check?

File: README.md [hunk 3]
```diff
@@ -75,11 +75,14 @@ Consult the [LORIS Wiki](https://github.com/aces/Loris/wiki/Setup) page on this
...
 
 If your apache configuration was not completed by the Install script, run the following enable rewriting of LORIS, enable your `$projectname` site, and restart apache:  (run by user who has root privileges)
-```bash
+    
+    ```bash
     sudo a2enmod rewrite
     sudo a2dissite default
     sudo a2ensite $projectname
     sudo service apache2 reload
+    ```
```
Comment (johnsaigle) [inline README.md]: These lines all need to be aligned all the way to the left and then it should work

Comment (christinerogers) [reply README.md]: got it.  done.

**Comment by christinerogers** [comment]
This should do it.  Renders cleanly here: https://github.com/christinerogers/Loris/blob/Fix_typo_formatting_main_Readme/README.md

**Pull request merged**